op-bindings/solc/types.go

```diff
@@ -64,6 +64,17 @@ func (s *StorageLayout) GetStorageLayoutEntry(name string) (StorageLayoutEntry,
 	return StorageLayoutEntry{}, fmt.Errorf("%s not found", name)
 }
 
+// GetStorageLayoutType returns the StorageLayoutType where the label matches
+// the provided name.
+func (s *StorageLayout) GetStorageLayoutType(name string) (StorageLayoutType, error) {
+	for typeName, typ := range s.Types {
+		if typeName == name {
+			return typ, nil
+		}
+	}
+	return StorageLayoutType{}, fmt.Errorf("%s not found", name)
```

Comment:
It looks like `s.Types` is a `map[string]StorageLayoutType`.
https://github.com/ethereum-optimism/optimism/blob/1403c7d7d66d11ceb517d2d05505c5b5a46e2882/op-bindings/solc/types.go#L53
Can we instead of looping over the map and checking equality of keys, just explicitly grab the item from the map as suggested below?
```suggestion
    if ty, ok := s.Types[name]; ok {
          return ty, nil
    }
	return StorageLayoutType{}, fmt.Errorf("%s not found", name)
```

Reply:
Yes that is definitely better, good catch